### ISSUE TITLE
Download local clients on Linux and Mac

### ISSUE THREAD

**Issue by Matvey16** [dbeaver/dbeaver#15093]
We need to find a way to download MySQL and PostgreSQL binaries needed for backup/restore tasks on Linux and Mac. We already do that for Windows, but users of other operating systems are very frustrated when they can't start a task and sometimes don't understand that those clients need to be installed manually.

**Comment by brooksvb** [comment]
Indeed, I encountered this issue on Manjaro Linux and was very annoyed that it needed to be done manually.
